shell/platform/tizen/channels/text_input_channel.cc

```diff
@@ -406,62 +260,37 @@ void TextInputChannel::SendStateUpdate(const TextInputModel& model) {
 bool TextInputChannel::FilterEvent(Ecore_Event_Key* event) {
   bool handled = false;
 
-#ifdef WEARABLE_PROFILE
+#ifdef __X64_SHELL__
+  bool is_ime = false;
+#elif WEARABLE_PROFILE
```

Comment:
I just noticed this is quite weird. Shouldn't it be like

```suggestion
#if defined(__X64_SHELL__)
  bool is_ime = false;
#elif defined(WEARABLE_PROFILE)
```